website/blog/2025-01-16-palladium.md

```diff
@@ -0,0 +1,234 @@
+---
+authors: kmarek
+title: Metals v1.4.3 - Palladium
+---
+
+We're happy to announce the release of Metals v1.4.3, which brings an array of bugfixes and improvements, as we keep working on Metals stability.
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">64</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">45</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">11</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">0</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: [https://github.com/scalameta/metals/milestone/75?closed=1](https://github.com/scalameta/metals/milestone/75?closed=1)
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs, Zed,
+Helix and Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from contributors from the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+
+- [Support for 2.13.16](#support-for-2.13.16)
+- [Fix for Metals stopped compiling bug](#fix-for-Metals-stopped-compiling-bug)
+- [Error reports improvements](error-reports-improvements)
+- [Inlay hints for worksheets](inlay-hints-for-worksheets)
+
+## Support for 2.13.16
+With the new release comes support for Scala 2.13.16. You can read release highlighs in [the Scala 2.13.16 release notes](https://github.com/scala/scala/releases/tag/v2.13.16).
```

Comment:
```suggestion
The new release comes with support for Scala 2.13.16. You can read release highlighs in [the Scala 2.13.16 release notes](https://github.com/scala/scala/releases/tag/v2.13.16).
```

Reply:
I feel like this change is more readable, since `comes` is close to `with` so it the complete phrase instead of mentalling having to connect them from afar